spec/presenters/utc_time_presenter_spec.rb

```diff
@@ -4,7 +4,7 @@
   describe '#to_s' do
     it 'returns the formatted timestamp in a string' do
       str = '2017-04-12 18:19:18 UTC'
-      timestamp = Time.parse(str)
+      timestamp = Time.zone.parse(str)
```

Comment:
Looks like this change broke this test.

Reply:
Yeah, I was gonna fix later. This PR is not critical to have today.